packages/uswds-core/package.json

```diff
@@ -14,19 +14,26 @@
     "u.s",
     "government"
   ],
-  "author": "18F",
-  "license": "ISC",
-  "main": "src/styles/_index.scss",
-  "directories": {
-    "test": "src/test"
+  "author": "uswds",
+  "license": "SEE LICENSE IN LICENSE.MD",
+  "exports": {
+    "./scss": "./_index.scss",
```

Comment:
This SCSS entrypoint would **only** be used when importing in a JS file. These don't apply when used in SCSS files.